tests/distribution_test.rs

```diff
@@ -0,0 +1,17 @@
+use bollard::{errors::Error, Docker};
+use common::*;
+use tokio::runtime::Runtime;
+
+#[macro_use]
+pub mod common;
+
+#[test]
+fn integration_test_inspect_registry_image() {
+    async fn inspect_test(docker: Docker) -> Result<(), Error> {
+        let image = create_image_hello_world(&docker).await?;
+        let _response = docker.inspect_registry_image(&image).await?;
+        Ok(())
```

Comment:
should I be validating each field here? I'm leaning towards no because I am trusting the generated openapi models

Reply:
No... just check something that's unlikely to change, like the `platform[].os` field.